### DISCUSSION TITLE
Indigenous Astronomy in Ottawa-Gatineau, Canada

### DISCUSSION THREAD

**Discussion by drpeterstockdale** [Stellarium/stellarium#1968] [Q&A]
I’m looking for help to verify some astronomical events on the hills in Ottawa-Gatineau, Canada.

I have been investigating the astronomical aspects of Ottawa-Gatineau, Canada from an indigenous perspective. I am particularly interested in determining whether Squaw/Cache Bay below UQAO might have been useful to see the **lunar standstill**. It was used by girls for coming of age. 

I have been clambering the hills of the area during solstices and equinoxes as I was told that Parliament Hill was used for solstices. 

I learned that Shirley's Bay used to have a federal solar observatory. It made me think that nearby Aylmer Island might have been used for observations. The island has been used for burials for at least 5,000 years. As part of a shoreline cleanup, this summer I went to the island and noted that the entire southern half of the sky was very well exposed, really up to the NW. I understand that the Pleiades moves around the sky in the NW. There are some notches in the hills there.

drpeterstockdale@gmail.com 613-852-4527

**Comment by gzotti** [comment]
Stellarium can help you in various ways with illustrating the astronomical part, from using accurate (surveyed) horizon panoramas (User Guide ch.7), the ArchaeoLines plugin (ch.13.10) to 3D sceneries (ch.14). But we must leave the ethnographical part (interviews?) to you.

**Comment by drpeterstockdale** [comment]
Hi Georg,
Oh wow, I didn't realize you had accurate horizon panoramas. So you can
identify lunar standstill variations at this location? The tricky bit here
is that for Squaw Bay it is not just the distant horizon that is the issue
but also the observer would have to move up and down a narrow bay with
hills on either side.
Thanks,
Peter

**Comment by gzotti** [comment]
The horizon lines/panoramas are as accurate as you can survey them :-) I have then developed the ArchaeoLines plugin with such standstill applications in mind. If fast changes in the foreground are relevant, you should then even go for a landscape model to be experienced in the Scenery3D plugin. If your indigenous people have their own constellations, you can then also illustrate that.

To see what's possible you may also want to have a look into: Zotti, G., Hoffmann, S. M., Wolf, A., Chéreau, F., & Chéreau, G. (2021). The Simulated Sky: Stellarium for Cultural Astronomy Research. Journal of Skyscape Archaeology, 6(2), 221–258. https://doi.org/10.1558/jsa.17822

**Comment by drpeterstockdale** [comment]
That’s great news, Georg. Thanks for giving me that link.
Peter

On Thu, Oct 14, 2021 at 4:07 PM Georg Zotti ***@***.***>
wrote:

> The horizon lines/panoramas are as accurate as you can survey them :-) I
> have then developed the ArchaeoLines plugin with such standstill
> applications in mind. If fast changes in the foreground are relevant, you
> should then even go for a landscape model to be experienced in the
> Scenery3D plugin. If your indigenous people have their own constellations,
> you can then also illustrate that.
>
> To see what's possible you may also want to have a look into: Zotti, G.,
> Hoffmann, S. M., Wolf, A., Chéreau, F., & Chéreau, G. (2021). The Simulated
> Sky: Stellarium for Cultural Astronomy Research. Journal of Skyscape
> Archaeology, 6(2), 221–258. https://doi.org/10.1558/jsa.17822
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Stellarium/stellarium/discussions/1968#discussioncomment-1479167>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWBTI3GMHYFIFIGTPMZGM3TUG42BLANCNFSM5F6J52BQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>
-- 
613-852-4527
